bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java

```diff
@@ -232,10 +232,27 @@ public Row readModifyWriteRow(ReadModifyWriteRowRequest request) {
   private ResultScanner<Row> streamRows(ReadRowsRequest request) {
     boolean isGet = request.getTargetCase() == ReadRowsRequest.TargetCase.ROW_KEY;
 
+    int streamingBufferSize = retryOptions.getStreamingBufferSize();
+
+    // Batch request size is more performant with a value of 1 for single row gets, while a higher
+    // number is more performant for scanning
+    int batchRequestSize;
+
     // If it's a long running scan, reserve the pool. There is a gRPC bug that makes the Channel
-    // unavailable for other streams. Reserve a channel for now. This can probably be removed once
-    // we use gRPC 0.10.0. See https://github.com/grpc/grpc-java/issues/1175 for more details
-    Channel channel = isGet ? channelPool : channelPool.reserveChannel();
+    // unavailable for other streams. Reserve a channel for now. Reservations can probably be
+    // removed once we use gRPC 0.10.0. See https://github.com/grpc/grpc-java/issues/1175 for more
+    // details
+    Channel channel;
+
+    if (isGet) {
+      batchRequestSize = 1;
+      streamingBufferSize = 10;
```

Comment:
Each result represents 1 column family.  There may be more than one column family per row.  10 < 32 which was there before.